lib/openstax/rescue_from/mute_listener.rb

```diff
@@ -0,0 +1,19 @@
+module OpenStax
+  module RescueFrom
+    class MuteListener
```

Comment:
I like the idea of the service class too because it would know everything to do so anyone with an adequate listener could use that service within the app that has the gem. Just have to define your before/after hooks for setup and response